### PR TITLE
Usage statistcs reporting trggered by FE client

### PR DESCRIPTION
**Pull request by gadomsky**

### CODE REVIEW

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/api/SettingsResources.scala [hunk 1]
```diff
@@ -95,7 +98,8 @@ object TopTabType extends Enumeration {
                                             tabs: Option[List[TopTab]],
                                             intervalTimeSettings: IntervalTimeSettings,
                                             testDataSettings: TestDataSettings,
-                                            redirectAfterArchive: Boolean
+                                            redirectAfterArchive: Boolean,
+                                            usageStatisticsReports: UsageStatisticsReportsSettings,
```
Comment (arkadius) [inline designer/server/src/main/scala/pl/touk/nussknacker/ui/api/SettingsResources.scala:102]: Do you know what is the rule for choosing whether we should add sth directly to `UISettings` or to `ToggleFeaturesOptions`. `AnalyticsConfig` we have on top level.

**Review by arkadius — APPROVED** [review]
LGTM, one confusion pointed.

File: designer/server/src/main/scala/pl/touk/nussknacker/ui/api/SettingsResources.scala (hunk 1 above)
Comment (gadomsky) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/api/SettingsResources.scala:102]: Unfortunetly I don't know.. maybe `analytics: Option[AnalyticsSettings]` should be moved to `ToggleFeaturesOptions` ?

Comment (gadomsky) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/api/SettingsResources.scala:102]: but it's not relatetd to this change, so can you e.g. create issue for later and resolve this comment?

Comment (arkadius) [reply designer/server/src/main/scala/pl/touk/nussknacker/ui/api/SettingsResources.scala:102]: Or everything should be flatten :D Let's leave it as it is ;)

**Pull request merged**